app/models/katello/pool.rb

```diff
@@ -2,14 +2,22 @@ module Katello
   class Pool < Katello::Model
     include Katello::Authorization::Pool
 
-    attr_accessor :quantity_attached
+    def quantity_attached
+      if activation_keys.present?
+        activation_keys.first.get_key_pools
+          .select { |pool| pool['id'] == cp_id }.first.try(:[], 'amount')
+      end
+    end
```

Comment:
@akofink that works if you can return the quantity attached for a specific activation key in the rabl file